patches/01_output_nodes_as_json.patch

```diff
@@ -140,7 +140,7 @@ index 0000000..eaded0f
 +static void
 +_outToken(StringInfo str, const char *s)
 +{
-+	if (s == NULL || *s == '\0')
++	if (s == NULL)
```

Comment:
Correct, that's a fix for #3. It was such a trivial change I just included it here ;)